x-pack/metricbeat/module/gcp/metrics/metrics_requester.go

```diff
@@ -152,7 +157,7 @@ func (r *metricsRequester) getFilterForMetric(serviceName, m string) string {
 			// }
 
 			region := strings.TrimSuffix(r.config.Region, "*")
-			f = fmt.Sprintf(`%s AND resource.labels.zone = starts_with("%s")`, f, region)
+			f = fmt.Sprintf(`%s AND resource.labels.region = starts_with("%s")`, f, region)
```

Comment:
Please do not include this change here, as it's not related to CloudSQL. This change is also a breaking change, so we need to take another approach.
I would add a comment, so we can remove this in the future: `FIXME: this is using zone but should use region. This is a breaking change.`